### PR TITLE
Fixed spelling, grammatical and formatting errors, and removed empty code box.

### PR DESCRIPTION
**Pull request by kjungwoo5**

### CODE REVIEW

File: tutorials/Atlas API usage.ipynb [hunk 1]
```diff
@@ -1273,15 +1273,15 @@
             "source": [
                 "---\n",
                 "**NOTE**: \n",
-                "the levels of the hierarchy depends on the underlying atlas, so we cannot ensure the goodness and consistency of their hierarchy three.\n",
+                "the levels of the hierarchy depend on the underlying atlas, so we cannot ensure the goodness and consistency of their hierarchy tree.\n",
```
Comment (adamltyson) [inline tutorials/Atlas API usage.ipynb:1276]:
```suggestion
                "The levels of the hierarchy depend on the underlying atlas, so we cannot ensure the goodness and consistency of their hierarchy tree.\n",
```

**Review by adamltyson — CHANGES_REQUESTED** [review]
Thanks @kjungwoo5, I didn't realise we had so many typos!

I've made a few suggestions. Most of these aren't actually corrections to your changes, but other things I noticed while reviewing this. Let me know what you think. 

You can just accept them via the pull request if you like.

**Comment by adamltyson** [comment]
Thanks @kjungwoo5. I think there's just one suggestion outstanding, then we're ready to go!

File: tutorials/Atlas API usage.ipynb [hunk 2]
```diff
@@ -211,7 +211,7 @@
             "cell_type": "markdown",
             "metadata": {},
             "source": [
-                "### 1.1 Anatomical, annotation and hemispheres stack"
+                "### 1.1 Anatomical, annotations and hemispheres stack"
```
[the remark this answers is not in the document]
Comment (kjungwoo5) [reply tutorials/Atlas API usage.ipynb:214]: I'll make changes in the subheadings below to reflect this change, then commit everything at once.

**Review by alessandrofelder — APPROVED** [review]

**Pull request merged**